mteb/evaluation/evaluators/normalize_encode.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
```

Comment:
I find the name of this module a bit confusing: the code doesn't really normalize embeddings right? It just converts them to an array. Maybe rename the module to `model_encode` just as the function or `encode`

Reply:
Wouldn't it be better to add the function to `Evaluator` class and inherit it in other child evaluators?